worlds/stardew_valley/__init__.py

```diff
@@ -171,15 +170,22 @@ def create_items(self):
                                for location in self.multiworld.get_locations(self.player)
                                if location.address is not None])
 
-        created_items = create_items(self.create_item, self.delete_item, locations_count, items_to_exclude, self.options, self.content,
-                                     self.random)
+        created_items = create_items(self.create_item, locations_count, items_to_exclude, self.options, self.content, self.random)
 
         self.multiworld.itempool += created_items
 
         setup_early_items(self.multiworld, self.options, self.player, self.random)
         self.setup_player_events()
         self.setup_victory()
 
+        # This is really a best-effort to get the total progression items count. The actual total can be impacted by the start_inventory_from_pool when items
+        # are removed from the bool, and probably when plando is applied as well... But since this is only used to spread grindy items across spheres, it's not
+        # a big deal if it's not 100% accurate.
+        self.total_progression_items += sum(1 for i in self.multiworld.precollected_items[self.player] if i.advancement)
+        self.total_progression_items += sum(1 for i in self.multiworld.get_filled_locations(1) if i.advancement)
```

Comment:
This takes a `player` arg, so passing in `1` seems very weird

Reply:
Good catch, this should be the player id

Reply:
Realized I could just make a suggestion
```suggestion
        self.total_progression_items += sum(1 for i in self.multiworld.get_filled_locations(self.player) if i.advancement)
```